lib/hunter.ex

```diff
@@ -333,6 +333,17 @@ defmodule Hunter do
   defdelegate unreblog(conn, id), to: Hunter.Status
 
   @doc """
+  Fetch the list of users who reblogged the status.
+
+  ## Parameters
+
+    * `conn` - connection credentials
+    * `id` - status identifier
+  """
+  @spec reblogged_by(Hunter.Client.t, non_neg_integer) :: Hunter.Status.t
+  defdelegate reblogged_by(conn, id), to: Hunter.Status
```

Comment:
I think this endpoint should return a list of accounts: `[Hunter.Account.t]` seems more appropriate according to your description.